tools/scaffold/template/datasource.tmpl

```diff
@@ -9,32 +9,32 @@ import (
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
 )
 
-const {{.NameCamelCase}}Name = "{{.NameSnakeCase}}" // TODO: if resource exists this can be deleted
```

Comment:
scaffolding adjustments, all of them can be seen in this PR and how a resource changes: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2679/commits/b9c320af268a76311129be5a97f9ccc3fec2e002

I can also move the scaffolding changes to a different PRs if that's preferred.